dracut-module-setup.sh

```diff
@@ -502,6 +502,29 @@ kdump_get_remote_ip() {
     echo "$_remote"
 }
 
+# Find the physical interface of Openvswitch (Ovs) bridge
+#
+# The physical network interface has the same MAC address as the Ovs bridge
+ovs_find_phy_if() {
+    local _mac _dev
+    _mac=$(</sys/class/net/$1/address)
```

Comment:
There's a kdump_get_mac_addr function doing the same thing. Although it's currently unused...